addon/-private/decorators/localized-attr.js

```diff
@@ -11,7 +12,22 @@ export default function (...args) {
   attrComputed.get = function () {
     const value = getter.call(this);
 
-    return value && value[this.getFieldLocale()];
+    if (!value) {
+      return;
+    }
+
+    const {
+      localizedModel: { showSomething = false, fallbacks = [] } = {}
+    } = getOwner(this).resolveRegistration('config:environment');
+
+    const keys = [this.getFieldLocale(), ...fallbacks];
+    let key = keys.find(key => Boolean(value[key]));
+
+    if (!key && showSomething) {
+      key = Object.keys(value).find(key => Boolean(value[key]));
+    }
+
+    return value[key];
```

Comment:
Maybe lets fail early to spare some execution and logic for the straight forward cases?:
```suggestion
    if(value[this.getFieldLocale()] || (!showSomething && !fallbacks.length)){
      return value[this.getFieldLocale()];
    }
    
    let key = fallbacks.find(key => Boolean(value[key]));

    if (!key && showSomething) {
      key = Object.keys(value).find(key => Boolean(value[key]));
    }

    return value[key];
```

Reply:
Done